cfme/control/explorer/alerts.py

```diff
@@ -256,7 +257,14 @@ def _fill(self, view):
             else:
                 fill_details["snmp_trap_send"] = True
                 fill_details["snmp_trap"] = self.snmp_trap
-        view.fill(fill_details)
+        if self.emails is not None:
+            if not self.emails:
```

Comment:
@izapolsk I think it shouldn't be changed. There is a difference between `self.emails = None` and `self.emails = []`. In the first case widgetastic will skip filling email at all, in the second it should uncheck the checkbox.